lib/js/src/manager/screen/utils/_VoiceCommandUpdateOperation.js

```diff
@@ -119,6 +135,31 @@ class _VoiceCommandUpdateOperation extends _Task {
         return this._errorArray.length === 0;
     }
 
+    /**
+     * Returns an array of VoiceCommands that are in the first array but not the second array
+     * @param {VoiceCommmand[]} firstArray - an array of VoiceCommands
+     * @param {VoiceCommand[]} secondArray - an array of VoiceCommands
+     * @returns {VoiceCommand[]} - An array of VoiceCommands that are in the first array but not the second array
+     */
+    _voiceCommandsNotInSecondArray (firstArray = null, secondArray = null) {
+        if (!Array.isArray(firstArray) || firstArray.length === 0) {
+            return secondArray;
```

Comment:
Elements from the second array should not be in the result according to the docs. Should return `[]` in this case
```suggestion
            return [];
```